release_templates/vintage/aws/v1alpha3/nodepools/awsmachinedeployment.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: infrastructure.giantswarm.io/v1alpha3
+kind: AWSMachineDeployment
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/ui-api/management-api/crd/awsmachinedeployments.infrastructure.giantswarm.io/
+  labels:
+    cluster.x-k8s.io/cluster-name: ${cluster_id}
+    giantswarm.io/cluster: ${cluster_id}
+    giantswarm.io/machine-deployment: ${machine_deployment_id}
+    giantswarm.io/organization: ${organization}
+    release.giantswarm.io/version: ${release}
+  name: ${machine_deployment_id}
+  namespace: org-${organization}
+spec:
+  nodePool:
+    description: General purpose
+    machine:
+      dockerVolumeSizeGB: 100
+      kubeletVolumeSizeGB: 100
+    scaling:
+      max: 1
+      min: 1
+  provider:
+    availabilityZones:
+    - eu-central-1a
```

Comment:
Needs to be templated or changed otherwise. Not helpful in a general base for AWS.